components/recurring-contributions/UpdateOrderPopUp.js

```diff
@@ -0,0 +1,327 @@
+                              value={inputAmountValue}
+                              onChange={setInputAmountValue}
+                              min={100}
+                              precision={3}
+                              px="2px"
+                            />
+                          </Box>
+                          <P fontSize="Caption" fontWeight="600" my={2}>
+                            <FormattedMessage
+                              defaultMessage="Min. amount: {minAmount}"
+                              id="RecurringContributions.minAmount"
+                              values={{
+                                minAmount: formatCurrency(minimumAmount, currency),
+                              }}
+                            />
+                          </P>
+                        </Flex>
+                      )}
+                    </Fragment>
+                  ) : (
+                    <Fragment>
+                      {flexible && (
+                        <P fontSize="Caption" fontWeight={400} lineHeight="Caption" color="black.500">
+                          <FormattedMessage id="ContributeTier.StartsAt" defaultMessage="Starts at" />
+                        </P>
+                      )}
+                      <P fontWeight={400} color="black.900">
+                        <FormattedMoneyAmount amount={amount} interval={interval.toLowerCase()} currency={currency} />
+                      </P>
+                    </Fragment>
+                  )}
+                </Flex>
+              </Flex>
+            </TierBox>
+          )}
+        </StyledRadioList>
+      )}
+      <Flex flexGrow={1 / 4} width={1} alignItems="center" justifyContent="center">
+        <Flex flexGrow={1} alignItems="center">
+          <StyledHr width="100%" />
+        </Flex>
+      </Flex>
+      <Flex flexGrow={1 / 4} width={1} alignItems="center" justifyContent="center" minHeight={45}>
+        <StyledButton
+          buttonSize="tiny"
+          onClick={() => {
+            setMenuState('mainMenu');
+          }}
+        >
+          {intl.formatMessage(messages.cancel)}
+        </StyledButton>
+        <StyledButton
+          ml={2}
+          buttonSize="tiny"
+          buttonStyle="secondary"
+          loading={loadingUpdateOrder}
+          onClick={async () => {
+            try {
+              await submitUpdateOrder({
+                variables: {
+                  order: { id: contribution.id },
+                  amount: selectedAmountOption.label === 'Other' ? inputAmountValue : selectedAmountOption.value,
+                  tier: {
+                    legacyId: selectedTier.value ? selectedTier.value.id : selectedTier.id,
+                    flexible: selectedTier.flexible,
+                    minimumAmount: selectedTier.minimumAmount,
+                  },
+                },
+                refetchQueries: [
+                  {
+                    query: getTiersQuery,
```

Comment:
Why do we need to refetch this one?

Reply:
It was refetched in an attempt to refresh the tiers and show the updated tier if you open the card again (which it currently does not). I need to read up on this more and have discussed Apollo cacheing with Francois so I think this will be fixed in an upcoming PR. Will remove it for now.